php/commands/search-replace.php

```diff
@@ -91,6 +91,13 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 *
 	 *     # Search/replace to a SQL file without transforming the database
 	 *     wp search-replace foo bar --export=database.sql
+	 * 
+	 *     # Search/replace production to development url (multisite compatible)
+	 *     if $(wp --url=http://example.com core is-installed --network); then
```

Comment:
Hmm. Maybe I could update the command to **Bash script: Search/replace production to development url (multisite compatible)** or add a `#!/bin/bash` at the start of it.

Any ideas?